apps/emqx_bridge_kafka/src/emqx_bridge_kafka.app.src

```diff
@@ -0,0 +1,15 @@
+{application, emqx_bridge_kafka, [
+    {description, "EMQX Enterprise Kafka Bridge"},
+    {vsn, "0.1.0"},
+    {registered, [emqx_bridge_kafka_consumer_sup]},
+    {applications, [
+        kernel,
+        stdlib,
+        emqx_ee_connector,
```

Comment:
A couple of concerns / questions.
1. Shouldn't we depend on `emqx_connector`? In the not so distant future `emqx_ee_connector` will be split into several different applications, and there'll be no `emqx_ee_connector` application as such. Also `emqx_ee_connector` doesn't depend on `emqx_connector` / `emqx_resource` which is kinda confusing.
2. Shouldn't we depend on the client libraries here (e.g. `wolff` / `brod`)? Also, there's no need to list them in `emqx_ee_connector.app.src`?

Reply:
1. I think I accidentally inherited the dependency on `emqx_ee_connector` from the ee_bridge app.  I tested locally, and apparently we don't need either connector app in the applications list.
2. Good catch!  Forgot to move those deps. :beers:

Reply:
> apparently we don't need either connector app in the applications list.

Just a small nit. Dependencies are transitive, so they might actually be used to simplify test setups (i.e. `application:ensure_all_loaded(emqx_ee_kafka)` instead of `application:load(emqx_connector), application:load(emqx_resource), ...`). Also, it would be harder to forget to also explicitly include [something in the release](https://github.com/emqx/emqx/blob/d16b5c40d658e8fbabc0774b53c1e615a9678eb6/rebar.config.erl#L375-L409).

Though I admit I might be missing some conventions on `app.src`'s dependencies.

Reply:
I'm double checking locally that the release and the test suites run smoothly with those changes.